### ISSUE TITLE
MyLittleTools

### ISSUE THREAD

**Issue by digitalhappens** [VCVRack/library#575]
Dear VCV Library maintainers
I am starting plugin development for vcv rack and I'd like to share my first plugin with one module at this time:
https://github.com/digitalhappens/MyLittleTools

Tested on Linux.

**Comment by cschol** [comment]
```
[MyLittleTools] Issues found in `plugin.json`:

Missing key: brand
Invalid license ID: TFPL
-- License must be a valid Identifier from https://spdx.org/licenses/
```

**Comment by digitalhappens** [comment]
I have changed / corrected the license...

**Comment by cschol** [comment]
Please only close the issue in your project. This issue is used for build tracking. Thank you.

**Comment by digitalhappens** [comment]
Hello I've just updated my plugin from 1.0.0 to version 1.0.1
Branch: mlt101
https://github.com/digitalhappens/MyLittleTools/tree/mlt101

I hope I've got everything right... tested on linux...

**Comment by digitalhappens** [comment]
Hello, a user informed me that there were missing wrong files... I've corrected that... the files in the svg-files res-folder changed... so do I need to count up my version number?

**Comment by cschol** [comment]
No. It has not been built yet and I have taken it out of the queue to review.

**Comment by digitalhappens** [comment]
thank you

**Comment by digitalhappens** [comment]
Hello I've just updated my plugin from 1.0.1 to version 1.0.2
Branch: mlt102
https://github.com/digitalhappens/MyLittleTools/tree/mlt102

I hope I've got everything right... tested on linux and windows...

**Comment by digitalhappens** [comment]
Hello I've just updated my plugin from 1.0.2 to version 1.0.3
Branch: mlt103
https://github.com/digitalhappens/MyLittleTools/tree/mlt103

I hope I've got everything right... tested on linux and windows...

**Comment by cschol** [comment]
Build failure on Windows platform: https://github.com/digitalhappens/MyLittleTools/issues/8

**Comment by digitalhappens** [comment]
I've changed the type to size_t it should now compile...

**Comment by AndrewBelt** [comment]
The Rack API has changed, making this not build with the Rack SDK 1.1.4. In particular, `plugin::allowedTags` has been removed, and the data structure you need now is `tag::tagAliases`. This is part of the "private" API of Rack since it is not included in `rack.hpp`, so it is subject to change. To fix, download that SDK version and fix the build.

**Comment by digitalhappens** [comment]
yeah I know, so I fixed that in branch mlt103 / version 1.0.3 of MyLittleTools... It should now work again...

**Comment by digitalhappens** [comment]
I've compiled this version on Linux and Windows with the latest Rack Version and it works...
